utils/podman/Makefile

```diff
@@ -40,21 +40,21 @@ define Package/podman/Default
   SECTION:=utils
   CATEGORY:=Utilities
   TITLE:=Podman
-  DEPENDS:=$(GO_ARCH_DEPENDS) +conmon +cni +cni-plugins +btrfs-progs +glib2 +gnupg2 +iptables +libgpg-error +libseccomp +libgpgme +nsenter +zoneinfo-simple +kmod-veth
   URL:=https://podman.io
 endef
 
 define Package/podman
   $(call Package/podman/Default)
+  DEPENDS:=$(GO_ARCH_DEPENDS) +conmon +cni +cni-plugins +btrfs-progs +glib2 +gnupg2 +iptables +libgpg-error +libseccomp +libgpgme +nsenter +zoneinfo-simple +kmod-veth
   CONFLICTS:=podman-selinux
   VARIANT:=default
   PROVIDES:=podman
 endef
 
 define Package/podman-selinux
   $(call Package/podman/Default)
-  TITLE+= with SELinux support
-  DEPENDS+= libselinux
+  TITLE += with SELinux support
+  DEPENDS:=$(GO_ARCH_DEPENDS) +conmon +cni +cni-plugins +btrfs-progs +glib2 +gnupg2 +iptables +libgpg-error +libseccomp +libgpgme +nsenter +zoneinfo-simple +kmod-veth +libselinux
```

Comment:
Why the previous issue was not okay for you?  While using `DEPENDS+=` there can be multiple dependencies.

Reply:
Sorry. My bad. My "private" version wasn't updated yet to that change which I made to version that I updated here earlier. Committing soon..

Reply:
@BKPepe fixed.